gamehub-frontend/src/components/Footer/Footer.css

```diff
@@ -9,6 +8,8 @@
     align-items: center;
     justify-content: center;
     min-height: 60px;
+    width: 100%;
+    box-sizing: border-box;
```

Comment:
I would probably set that for all elements so you don't have to bother with it again, so like
```
* {
    box-sizing: border-box;
}
```